build.rs

```diff
@@ -13,4 +13,38 @@ fn main() {
     {
         println!("cargo::rustc-cfg=feature=\"include-ui\"");
     }
+    match std::fs::File::open("assets/icon.png") {
+        // clippy complains about 'file' being unused, still functions as intended
+        Ok(_file) => {
+            if env::var_os("CARGO_CFG_WINDOWS").is_some() {
```

Comment:
I would move this `if` statement to be outside the entire `match` so it doesn't even begin checking icon stuff unless it is Windows.

(i.e.)
```rust
if env::var_os("CARGO_CFG_WINDOWS").is_some() {
    match std::fs::File::open("assets/icon.png") {
        Ok(_file) => {
            ...
        },
        ...
    }
}
```